### DISCUSSION TITLE
Custom Constraint: Visit one of the two nodes only

### DISCUSSION THREAD

**Discussion by mertbakir** [google/or-tools#2596] [Routing (and legacy CP) questions]
I want to add a custom constraint to [CVRPTW](https://github.com/google/or-tools/blob/master/ortools/constraint_solver/samples/cvrptw.py). Let's say I have 2 nodes and I want only one of them to be visited.
So, whichever of them is visited, other must be dropped without cost.

One step further, let's say we have n(5) nodes and I want **at_least** k (3) of them to be visited. 
I couldn't figure out how to implement such constraints. Any help is appreciated.

**Comment by Mizux** [comment]
did you try to use `routing.AddDisjunction([index_1, index_2,...], penalty, 3)` ?
